src/iMatix.xml

```diff
@@ -0,0 +1,68 @@
+  <urls>
+    <url>http://legacy.imatix.com/html/sfl/sfl4.htm#license</url>
+  </urls>
+  <license>
+    <title>
+      <p>The SFL License Agreement</p>
+    </title>
+    <body>
+      <p>This license agreement covers your use of the iMatix STANDARD FUNCTION LIBRARY (SFL), its source code,
+         documentation, and executable files, hereinafter referred to as "the Product".</p>
+      <p>The Product is Copyright (c) 1991-2000 iMatix Corporation. You may use it and distribute it according to
+         this following License Agreement. If you do not agree with these terms, please remove the Product from
+         your system. By incorporating the Product in your work or distributing the Product to others you
+         implicitly agree to these license terms.</p>
+      <p>Statement Of Copyright</p>
+      <p>The Product is, and remains, Copyright © 1991-2000 iMatix Corporation, with exception of specific
+         copyrights as noted in the individual source files.</p>
+      <p>Conditions Of Use</p>
+      <p>You do not need to provide the source code for the Product as part of your product. However, you must do
+         one of these things to comply with the Product License Agreement:</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Provide the source code for Product modules that you use, or</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Make your product freely available according to a license similar to the GNU General Public
+             License, or the Perl Artistic License, or</p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>Add this phrase to the documentation for your product: "This product uses parts of the iMatix
+             SFL, Copyright © 1991-2000 iMatix Corporation &lt;http://www.imatix.com&gt;".</p>
+        </li>
+      </list>
+      <p>Rights Of Usage</p>
+      <p>You may freely and at no cost use the Product in any project, commercial, academic, military, or private,
+         so long as you respect the License Agreement. The License Agreement does not affect any software
+         except the Product. In particular, any application that uses the Product does not itself fall under
+         the License Agreement.</p>
+      <p>You may modify any part of the Product, including sources and documentation, except this License
+         Agreement, which you may not modify.</p>
+      <p>You must clearly indicate any modifications at the start of each source file. The user of any modified
+         Product code must know that the source file is not original.</p>
+      <p>At your discretion, you may rewrite or reuse any part of the Product so that your derived code is not
+         obviously part of the Product. This derived code does not fall under the Product License Agreement
+         directly, but you must include a credit at the start of each source file indicating the original
```

Comment:
discussed on legal call; will add acknowledgement label to this PR so that we can make these conditional acknowledgement requirements extractable later